packages/forms/src/UIForm/utils/properties.test.js

```diff
@@ -55,4 +55,28 @@ describe('Properties utils', () => {
 			});
 		});
 	});
+
+	describe('#convertValue', () => {
+		it('should return the string value', () => {
+			// given
+			const value = '3';
+
+			// when
+			const convertedValue = convertValue('string', value);
```

Comment:
As number is the only type converted, i'm not sure this test is not misleading as it's now